### PR TITLE
Refactor/activity files

### PR DESCRIPTION
**Pull request by emilyjmiles**
## Type of change

- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)
- [x] Refactor

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

- [x] I tested my changes in the browser
- [x] I tested with Mocha/Chai

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new bugs
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Is your code D.R.Y?
- [x] Does it follow SRP?

### CODE REVIEW

File: src/Activity.js [hunk 1]
```diff
@@ -5,24 +5,54 @@ class Activity {
...
+  };
+
+  getDailyMilesByDate(date) {
+    const totalMiles = this.usersActivity.reduce((total, data) => {
+      const mileInFeet = 5280;
+      if (data.date === date) {
+        total = this.currentUser.strideLength * data.numSteps / mileInFeet
+      };
+
+      return total
```
Comment (ohClaire) [inline src/Activity.js:26]: semicolon here

File: src/Activity.js [hunk 2]
```diff
@@ -5,24 +5,54 @@ class Activity {
...
+      return 'This date could not be found.';
+    };
+
+  };
+
+  getDailyMilesByDate(date) {
+    const totalMiles = this.usersActivity.reduce((total, data) => {
+      const mileInFeet = 5280;
+      if (data.date === date) {
+        total = this.currentUser.strideLength * data.numSteps / mileInFeet
```
Comment (ohClaire) [inline src/Activity.js:23]: semicolon here

Comment (ohClaire) [reply src/Activity.js:23]: Awesome job on Activity! Excited to see these charts :)

Comment (emilyjmiles) [reply src/Activity.js:23]: Will add on the next push up

**Pull request merged**